VideoLocker/src/main/java/org/edx/mobile/module/notification/DummyNotificationDelegate.java

```diff
@@ -0,0 +1,37 @@
+package org.edx.mobile.module.notification;
+
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+
+import java.util.List;
+
+public class DummyNotificationDelegate implements NotificationDelegate{
+    @Override
+    public void unsubscribeAll() {
+
+    }
+
+    @Override
+    public void resubscribeAll() {
+
+    }
+
+    @Override
+    public void syncWithNotificationServer() {
+
+    }
+
+    @Override
+    public void checkCourseEnrollment(List<EnrolledCoursesResponse> responses) {
+
+    }
+
+    @Override
+    public void changeNotificationSetting(String courseId, boolean subscribe) {
+
+    }
+
+    @Override
+    public void subscribeAndUnsubscribeToNotificationServer(String channel, boolean subscribe) {
```

Comment:
Can we think of a better name for this? 
It looks like it toggles the subscription but the variable name seems to suggest that we subscribe and then unsubscribe.